test/inference.jl

```diff
@@ -494,3 +494,9 @@ immutable MyType18457{T,F,G}<:AbstractMyType18457{T,F,G} end
 tpara18457{I}(::Type{AbstractMyType18457{I}}) = I
 tpara18457{A<:AbstractMyType18457}(::Type{A}) = tpara18457(supertype(A))
 @test tpara18457(MyType18457{true}) === true
+
+# Issue 20067
+@generated function foo20067{T <: Tuple}(::Type{T}, elements::Tuple)
+           :(T)
+end
+code_typed(DevNull, foo20067, (DataType, Tuple{Int, Int}))
```

Comment:
Sorry about, that, I noticed that and then promptly forgot to fix it...
It might be best to remove the test for now. #20146